dataactcore/scripts/databaseSetup.py

```diff
@@ -41,3 +43,5 @@ def runMigrations(alembicDbName):
         if "relation" and "already exists" in e.message:
```

Comment:
That's strange. I think I pushed a change to this bit of code that checks the errorType instead of looking in e.message (in response to your comments in a code review). Is this a victim of the repo merge?